renderer-macos/lib/Cocoa.re

```diff
@@ -168,22 +168,49 @@ module NSTextView = {
 
   [@noalloc] external make: unit => t = "ml_NSTextView_make";
 
-  [@noalloc] external setStringValue: (t, string) => unit = "ml_NSTextView_setStringValue";
+  [@noalloc]
```

Comment:
I'd say we should create separate modules like `NSTextView` to tidy this up. Same goes for C stubs.

Reply:
You mean separate `NSTextView.re` file, not in `Cocoa`?

Reply:
yes

Reply:
Btw, I think the naming should be consistent. If we bind to BriskView or BRISKView, let's call the stub file the same.

Reply:
I'll keep modules in Cocoa for now, we'll split them apart in a different iteration.